### ISSUE TITLE
Question about TSDAE loss

### ISSUE THREAD

**Issue by boscoj2008** [UKPLab/sentence-transformers#1542]
Hi Nils. I have the following concerns;

(1) Could you please explain what TSDAE does to the embedding space? Given that contrastive learning aims to put similar examples close together in the embedding space than dissimilar examples it is not clear to me what TSDAE is doing to the embedding space.

(2) When pre-training with TSDAE, do we use the cross-entropy loss? The TSDAE looks like a cross-entropy loss at a high level unless I stand to be corrected.

Thank you

**Comment by nreimers** [comment]
1) It is a denoising auto encoder. So it tries to store all relevant information in the embedding so that you can re-construct the original text from it. It does not have such a clear vector space meaning as contrastive training as it can encode non-linear relationships.

2) We have a decoder, that predicts every word. For every word, CE loss is used

**Comment by boscoj2008** [comment]
I see. 

1. So in this case, I can think of TSDAE as a strategy to learn a transformer encoder (similar to MLM and NSP) using TSDAE loss. This encoder is what I can use for my downstream tasks by feeding the [CLS] token to a linear classifier? 

2. And will the encoder still support the usual input strategy; [CLS] sentence [SEP] sentence [SEP] in the downstream task?
